### PR TITLE
glasfaser page

### PR DESCRIPTION
**Pull request by tiwariav**

### CODE REVIEW

**Comment by telmich** [comment]
@pcoder I will merge it for now -- please carefully review and fix in coordination with @tiwariav later, where needed.

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -173,6 +173,11 @@ msgstr "Anmelden"
 msgid "Create VM"
 msgstr "VM erstellen"
 
+#, fuzzy
```
Comment (pcoder) [inline hosting/locale/de/LC_MESSAGES/django.po]: This translation seems to be for "Your VM hosted in Switzerland"

**Comment by tiwariav** [comment]
datacenterlight/static/datacenterlight/img/header-nico-sans-Recovered.jpg is removed in #519

File: datacenterlight/views.py [hunk 2]
```diff
@@ -55,7 +55,9 @@ def form_valid(self, form):
                 sender=form.cleaned_data.get('email')
             ),
             'from_email': settings.DCL_SUPPORT_FROM_ADDRESS,
-            'to': ['info@ungleich.ch'],
+            'to': '{}@ungleich.ch'.format(
```
[the remark this answers is not in the document]
Comment (tiwariav) [reply datacenterlight/views.py]: absolutely not. will change in #519

File: hosting/locale/de/LC_MESSAGES/django.po (hunk 1 above)
Comment (tiwariav) [reply hosting/locale/de/LC_MESSAGES/django.po]: fixed in #519

**Pull request merged**